nomad/structs/structs_codegen.go

```diff
@@ -1,3 +1,3 @@
 package structs
 
-//go:generate codecgen -d 100 -o structs.generated.go structs.go
+//go:generate ./generate.sh
```

Comment:
shouldn't there be a similar change to the directive in client/structs/structs.go